src/main/java/org/gridsuite/shortcircuit/server/entities/FeederResultEntity.java

```diff
@@ -6,20 +6,34 @@
  */
 package org.gridsuite.shortcircuit.server.entities;
 
-import lombok.AllArgsConstructor;
 import lombok.Getter;
 import lombok.NoArgsConstructor;
 
 import jakarta.persistence.*;
 
+import java.util.UUID;
+
 /**
  * @author Nicolas Noir <nicolas.noir at rte-france.com>
  */
 @Getter
-@AllArgsConstructor
 @NoArgsConstructor
-@Embeddable
-public class FeederResultEmbeddable {
+@Entity
+@Table(name = "feeder_results",
+        indexes = {@Index(name = "feeder_results_fault_result_idx",
+                columnList = "fault_result_entity_fault_result_uuid")})
+public class FeederResultEntity {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private UUID feederResultUuid;
+
+    @ManyToOne(
+            cascade = CascadeType.ALL,
+            fetch = FetchType.LAZY
+    )
+    @JoinColumn(name = "fault_result_entity_fault_result_uuid")
+    private FaultResultEntity faultResult;
```

Comment:
Maybe add a comment because we don't really need this bidirectional relation but it is the best way to do it to reduce query numbers in BDD and all like explain in https://vladmihalcea.com/the-best-way-to-map-a-onetomany-association-with-jpa-and-hibernate/

> Bidirectional @OneToMany associations are way better than unidirectional ones because they rely on the @ManyToOne relationship, which is always efficient in terms of generated SQL statements.

Reply:
Ok done